jspwiki-main/src/main/java/org/apache/wiki/ui/Installer.java

```diff
@@ -86,6 +86,7 @@ public Installer( final HttpServletRequest request, final ServletConfig config )
         // Stash the request
         m_request = request;
         m_validated = false;
+        TMP_DIR = m_engine.getWikiProperties().getProperty( "java.io.tmpdir" );
```

Comment:
should be `m_engine.getWikiProperties().getProperty( "jspwiki.workDir" )`

Reply:
done.